### DISCUSSION TITLE
"WizardLM 13B Uncensored" conversion to GGML

### DISCUSSION THREAD

**Discussion by pugzly** [ggml-org/llama.cpp#1394] [Q&A] [answered]
I just noticed "WizardLM 13B Uncensored" appeared in the wild
https://huggingface.co/ehartford/WizardLM-13B-Uncensored 

But I'm still trying to work out the correct process of conversion for "pytorch_model.bin" in to GGML
So I figured I'll check with guys around, if somebody here already done it and has all the right steps at hand?

(while I continue reading through all docs and experiment)

EDIT:
Thanks to Geen-SKY, it was as simple as:
`
python3 convert.py models/WizardLM-13B-Uncensored/pytorch_model.bin
`
and
`
./quantize ./models/WizardLM-13B-Uncensored/pytorch_model.bin ./models/WizardLM-13B-Uncensored/ggml-model-q4_0.bin q4_0
`

**Comment by Green-Sky** [comment, marked as answer]
Simply run the `convert.py` pointing to the directory you put the files.
then run the `qunatize` executable for the desired quantization.
done.

**Reply by Green-Sky** [reply]
more details in the readme https://github.com/ggerganov/llama.cpp#prepare-data--run

**Reply by pugzly** [reply]
ah yes, of course, it had to be something painfully simple..
I didn't realize I had to provide full path to bin file for conversion. Thanks a lot!

For anyone else passing by, I only needed run the following:
python3 convert.py models/WizardLM-13B-Uncensored/pytorch_model.bin
./quantize ./models/WizardLM-13B-Uncensored/pytorch_model.bin ./models/WizardLM-13B-Uncensored/ggml-model-q4_0.bin q4_0

**Reply by Green-Sky** [reply]
using just `python3 convert.py models/WizardLM-13B-Uncensored/` was not enough?

**Reply by pugzly** [reply]
no, it was throwing me error, that there was no model in that directory, until I gave it full path to the file
